howso/engine/trainee.py

```diff
@@ -1686,6 +1687,11 @@ def react_series(
             See parameter ``desired_conviction`` in :func:`react`.
         details : dict of {str: object}
             See parameter ``details`` in :func:`react`.
+        exclude_novel_nominals_from_uniqueness_check : bool, default False
+            exclude_novel_nominals_from_uniqueness_check : bool, default False
+            If True, will exclude features which have a subtype defined in their feature
```

Comment:
"feature" is duplicated in this sentence here and in other instances where this parameter is defined.